static/scss/user-chip.scss

```diff
@@ -32,6 +32,7 @@
 
     a:hover {
       text-decoration: none !important;
+      cursor: pointer;
```

Comment:
If this is only because of the `<a onClick>` element referenced above, then after changing it to a `<button>` element, this line should be removed. (Or is there some other reason why this was added?)